src/components/Overlays/Modal/Modal.tsx

```diff
@@ -45,6 +45,9 @@ export interface ModalProps extends Omit<HTMLAttributes<HTMLDivElement>, 'onAnim
   snapPoints?: (number | string)[];
   /** Snap point index at which the overlay begins to fade, enhancing visual cues for modal depth. */
   fadeFromIndex?: never;
+  /** makes the modal window not close */
```

Comment:
```suggestion
  /** Makes the modal window not close */
```

Reply:
/** Determines if the modal can be closed by user interactions such as clicking outside or pressing the escape key. */